packages/templates/clients/websocket/javascript/components/AvailableOperations.js

```
@@ -0,0 +1,29 @@
+import { Text } from '@asyncapi/generator-react-sdk';
+import MessageExamples from './MessageExamples';
+import OperationHeader from './OperationHeader';
+
+export function AvailableOperations({ operations }) {
+  return (
+    <>
+      <Text newLines={2}>### Available Operations</Text>
+      {operations.length > 0 ? (
+        operations.map((operation) => (
+          <Text newLines={2}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)